packages/database/src/migrations/20210927003701-LESMIS5YearOldEnrolmentRateMapOverlays-modifies-data.js

```diff
@@ -0,0 +1,201 @@
+'use strict';
+
+import { generateId, insertObject, codeToId, nameToId, arrayToDbString } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const PERMISSION_GROUP = 'LESMIS Public';
+const OVERLAY_CODE = 'LESMIS_enrolment_ece';
+
+const OVERLAY_GROUPS = [
+  {
+    parentCode: 'Root',
+    children: [
+      {
+        name: 'ECE/Pre-Primary Enrolment',
+        code: `${OVERLAY_CODE}_Group`,
+      },
+    ],
+  },
+  {
+    parentCode: `${OVERLAY_CODE}_Group`,
+    children: [
+      {
+        name: 'District Level',
+        code: `${OVERLAY_CODE}_District_Group`,
+      },
+    ],
+  },
+];
```

Comment:
I think we can get rid of `OVERLAY_GROUPS` completely, the groups already exist from another migration, so this is just muddying the waters a bit